src/rpc/rawtransaction.cpp

```diff
@@ -329,8 +339,12 @@ static UniValue gettxchainlocks(const JSONRPCRequest& request)
         if (height != -1) {
             chainLock = llmq_ctx.clhandler->HasChainLock(height, hash_block);
         }
+        else {
+            result.pushKV("mempool", true);
+        }
```

Comment:
probably a typo and was meant to be `mempool = true;` but better
```suggestion
```
and see other suggestions